src/osemosys/schemas/model.py

```diff
@@ -22,22 +28,24 @@ class RunSpec(OSeMOSYSBase):
     commodities: List[Commodity]
 
     # impact constraints (e.g. CO2)
-    impacts: List[Impact]
+    #TODO
+    #impacts: List[Impact]
 
     # technologies
-    production_technologies: List[TechnologyProduction]
-    storage_technologies: List[TechnologyStorage]
-    transmission_technologies: List[TechnologyTransmission]
+    #TODO
+    #production_technologies: List[TechnologyProduction]
+    #storage_technologies: List[TechnologyStorage]
+    #transmission_technologies: List[TechnologyTransmission]
 
     # reserve margins if any
-    reserve_margins_commodity: List[List[str,RegionYearData]] | None
-    reserve_margins_technology: List[List[str, RegionalYearData]] | None
+    reserve_margins_commodity: List[Dict[str,RegionYearData]] | None
+    reserve_margins_technology: List[Dict[str, RegionYearData]] | None
 
     # renewable targets
     renewable_targets: RegionYearData | None
     
 
-    def to_simplicity(self, root_dir) -> Dict[str,str]:
+    def to_simplicity(self, comparison_directory) -> Dict[str,str]:
```

Comment:
we want to keep this generalised - we won't always be using this for comparison necessarily